abstract/myst.yml

```diff
@@ -44,10 +51,10 @@ project:
 site:
   template: article-theme
   actions:
-    # - title: pdf
-    #   url: ARTICLE-NAME.pdf
-    #   static: true
-    # - title: slides
-    #   url: presentation/SLIDES-NAME.pdf
-    #   static: true
+    - title: poster
+      url: presentation/AGU2023_Jorge_eposter.pdf
+      static: true
```

Comment:
Since this was a talk, we don't need to include an eposter. Would you mind removing this line and the e_poster pdf? Thanks @tetikshajain!